test/server/graphql/v1/mutation.test.js

```diff
@@ -13,6 +14,9 @@ import * as utils from '../../../utils';
 let host, user1, user2, user3, collective1, event1, ticket1;
 let sandbox, executeOrderStub, emailSendSpy, emailSendMessageSpy;
 
+chai.use(chaiThings);
```

Comment:
chai plugins are added globally in `test/setup.js` to make them available in all tests.

Reply:
I realize that I should have added package-lock to the commit. My bad

Reply:
Yep, but this comment still stands :wink:

Reply:
just fixed it